pkg/tests/service/options.go

```diff
@@ -143,6 +145,17 @@ func (opt *Options) validate() {
 	}
 }
 
+// BuildHAKeeperConfig returns hakeeper.Config
+//
+// We could check timeout for dn/log store via hakeeper.Config.
+func (opt Options) BuildHAKeeperConfig() hakeeper.Config {
+	return hakeeper.Config{
+		TickPerSecond:   opt.hakeeper.tickPerSecond,
+		LogStoreTimeout: opt.hakeeper.logStoreTimeout,
+		DnStoreTimeout:  opt.hakeeper.dnStoreTimeout,
```

Comment:
Use DNStoreTimeout

Reply:
DnStoreTimeout is a field of `hakeeper.Config` which is defined within package `github.com/matrixorigin/matrixone/pkg/hakeeper`.